src/makeRobot.js

```diff
@@ -38,7 +38,57 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    name,
+    wheels,
+    version,
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(num = 1) {
+      if (num > 0) {
+        this.coords.y += num;
+      }
+
+      return this;
+    },
+
+    goBack(num = 1) {
+      if (num > 0) {
+        this.coords.y -= num;
+      }
+
+      return this;
+    },
+
+    goRight(num = 1) {
+      if (num > 0) {
+        this.coords.x += num;
+      }
+
+      return this;
+    },
+
+    goLeft(num = 1) {
+      if (num > 0) {
+        this.coords.x -= num;
+      }
+
+      return this;
+    },
+    evacuate() {
```

Comment:
empty line to separate method
```suggestion
    },
    
    evacuate() {
```